### PR TITLE
Contributors page refresh #2594

### PR DESCRIPTION
**Pull request by ccanos**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/core/i18n/es/translation.es.json [hunk 1]
```diff
@@ -277,6 +277,10 @@
         "QUESTIONS": "Preguntas",
         "SHARING": "Compartir"
       }
+    },
+    "blocks": {
+      "users-contributing": "los usuarios que están contribuyendo.",
+      "all-contributing-users": "todos los colaboradores."
     }
```
Comment (alstojanovic) [inline src/core/i18n/es/translation.es.json:284]: This needs to be added to nl, bg and ua translations.

File: src/core/i18n/es/translation.es.json [hunk 2]
```diff
@@ -909,19 +914,15 @@
     },
     "contributors": {
       "search": {
-        "title": "¡Explora los usuarios y organizaciones que están contribuyendo!",
-        "subtitle": "Introduce algún término de búsqueda. Haz click sobre las tarjetas para ver los perfiles de los resultados.",
-        "placeholder": "Buscar usuarios u Organizaciones"
+        "title": "Mira quienes están contribuyendo",
+        "subtitle": "Echa un ojo a las organizaciones y usuarios que están contribuyendo a los Desafíos, Oportunidades y Hubs en la plataforma",
+        "placeholder": "Buscar..."
```
Comment (alstojanovic) [inline src/core/i18n/es/translation.es.json:919]: These probably need to be updated for nl, bg and ua.

File: src/core/i18n/es/translation.es.json [hunk 3]
```diff
@@ -470,7 +474,8 @@
     },
     "backdrop": {
       "private": "Por favor, únete para acceder a {{blockName}}",
-      "authentication": "Por favor, inicia sesión para ver los detalles sobre {{blockName}}."
+      "authentication": "Por favor, inicia sesión para ver los detalles sobre {{blockName}}.",
+      "login": "Por favor, inicia sesión para ver {{blockName}}."
```
Comment (alstojanovic) [inline src/core/i18n/es/translation.es.json:478]: Need to be added for nl, bg and ua.

**Review by alstojanovic — CHANGES_REQUESTED** [review]
I added questions, nothing needs explicit change except we are missing some translations for nl, bg and ua languages.

File: src/domain/shared/components/Backdrops/ImageBackdrop.tsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+}));
+
+const FlexSpacer = styled(Box)(() => ({
+  flexBasis: '50%',
+}));
+
+interface ImageBackdropProps extends BackdropProps {
+  src: string;
+  backdropMessage: 'private' | 'authentication' | 'login'; // components.backdrop.* in the translation
+  blockName: 'users-contributing' | 'all-contributing-users'; // common.block.* in the translation
```
[the remark this answers is not in the document]
Comment (ccanos) [reply src/domain/shared/components/Backdrops/ImageBackdrop.tsx:46]: I think they are useful... 
I have changed them to 
image
I like to have this, it helps when adding a new block

File: src/domain/community/contributor/ContributorsView.tsx [hunk 5]
```diff
@@ -1,102 +1,169 @@
...
+            <Grid container spacing={1}>
+              {users?.loading &&
+                times(ITEMS_PER_PAGE, i => (
+                  <Grid item flexBasis={`${GRID_ITEM_WIDTH}%`} key={i}>
+                    <ContributorCardSkeleton />
+                  </Grid>
+                ))}
+              {!users?.loading &&
+                users?.items?.map(userToContributorCard).map((user, i) => (
+                  <Grid item flexBasis={`${GRID_ITEM_WIDTH}%`} key={i}>
```
[the remark this answers is not in the document]
Comment (ccanos) [reply src/domain/community/contributor/ContributorsView.tsx:139]: Good point, we talked about this yesterday. Fixed

File: src/core/i18n/es/translation.es.json (hunk 2 above)
Comment (techsmyth) [reply src/core/i18n/es/translation.es.json:919]: We need to address translations structurally. These can be left out for this version, they will default to English.

**Review by alstojanovic — APPROVED** [review]

**Pull request merged**